gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.10-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.14.1-all.zip
```

Comment:
I don't have access to our server (it's asking me for credentials).

What mismatch are you talking about exactly ? The android sdk is fully updated on the build agent and I installed the latest version of Gradle Jenkins plugin and the job is using Gradle 2.14.1.